src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -75,7 +75,7 @@ const styles = StyleSheet.create({
     fontSize: normalizeFontSize(16),
     fontFamily: fonts.inter[600],
     textAlign: 'left',
-    color: isHeaderHide ? COLORS.white : COLORS.black,
+    color: isHeaderHide ? COLORS.white2 : COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been changed from `COLORS.white` to `COLORS.white2` when `isHeaderHide` is true. Similar to the previous comment, ensure that `COLORS.white2` is a valid and defined color in your project's color palette, and it provides the desired visual effect on different screens and themes (light/dark mode).
```diff
- color: isHeaderHide ? COLORS.white : COLORS.black,
+ color: isHeaderHide ? COLORS.white2 : COLORS.black,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->